bin/saya/src/args/mod.rs

```diff
@@ -73,6 +64,10 @@ pub struct SayaArgs {
     #[command(flatten)]
     #[command(next_help_heading = "Choose the proof pipeline configuration")]
     pub proof: ProofOptions,
+
+    #[command(flatten)]
+    #[command(next_help_heading = "Starknet account configuration")]
```

Comment:
```suggestion
    #[command(next_help_heading = "Starknet account configuration for settlement.")]
```
I'm not sure I proposed the best wording here, but it could be great to have something short that actually mention something related to the settlement.
Or we may have something else than `next_help_heading` where we may put more information, wdyt?